lfm/templates/loved.html

```diff
@@ -1,13 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<rss version="2.0" xmlns:dc="http://purl.org/dc/elements/1.1/">
+<rss version="2.0" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:lfm="https://www.last.fm/api">
 	<channel>
 		<title>{{context.user}} - loved tracks</title>
         <lastBuildDate>{{ ''|rfc822_date() }}</lastBuildDate>
         <language>en</language>
-		<link>https://lfm.xiffy.nl/{{context.user}}/{{context.type}}</link>
+        <link>https://www.last.fm/user/{{context.user}}/loved</link>
```

Comment:
I disagree with this line. The <link> tag inside RSS should point to self, so that means the original line of code.

Reply:
i stand corrected from the rss-spec:
The URL to the HTML website corresponding to the channel.